### PR TITLE
[2.11.x] DDF-3590 Update jackson libraries to 2.9.4

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Upgrates the jackson libraries to 2.9.4
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@emmberk @peterhuffer 
#### Choose 2 committers to review/merge the PR. 
@bdeining
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)
CI Build passes

#### What are the relevant tickets?
[DDF-3590](https://codice.atlassian.net/browse/DDF-3590)

### CODE REVIEW

**Comment by clockard** [comment]
@bdeining I think it is because the fix version is so new

**Comment by bdeining** [comment]
@clockard i wonder if we should have a ticket to remove the suppression once that's fixed; i'd be concerned that this would be suppressed in the future when it shouldn't be

**Comment by bdeining** [comment]
build now

File: pom.xml [hunk 1]
```diff
@@ -164,6 +164,7 @@
         <jodatime2-bundle.version>2.9.9</jodatime2-bundle.version>
         <aws-java-sdk-bundle.version>1.11.125_1</aws-java-sdk-bundle.version>
         <javax-mail.version>1.4.7</javax-mail.version>
+        <jackson2-version>2.9.4</jackson2-version>
```
Comment (shaundmorris) [inline pom.xml]: ❓ Why the `2` in `jackson2`?

**Comment by clockard** [comment]
build now

File: pom.xml (hunk 1 above)
Comment (clockard) [reply pom.xml]: That was the variable in use before.

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**